src/odemis/gui/cont/streams.py

```diff
@@ -2321,7 +2321,6 @@ def _addRepStream(self, stream, mdstream, vas, axes, **kwargs):
         # present. (and get the controls via stream_cont.entries[vaname])
         # Add VAs (in same order as config)
         vas = util.sorted_according_to(vas, stream_config.keys())
```

Comment:
You can strip even more out. In particular, you can remove the `vas` argument, and from there, simplify the all the callers.